pkgs/development/python-modules/pytest-xdist/default.nix

```diff
@@ -1,30 +1,27 @@
-{ stdenv, fetchPypi, buildPythonPackage, execnet, pytest
+{ stdenv, fetchPypi, buildPythonPackage, execnet, pytest_6
 , setuptools_scm, pytest-forked, filelock, psutil, six, isPy3k }:
 
 buildPythonPackage rec {
   pname = "pytest-xdist";
   version = "2.1.0";
+  disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "82d938f1a24186520e2d9d3a64ef7d9ac7ecdf1a0659e095d18e596b8cbd0672";
+    sha256 = "0wh6pn66nncfs6ay0n863bgyriwsgppn8flx5l7551j1lbqkinc2";
```

Comment:
why the hash change?

Reply:
The diff is deceiving. I did not write this but rather renamed 2.nix to default.nix.